samples/sample_common/src/sample_utils.cpp

```diff
@@ -992,36 +1046,34 @@ mfxStatus CSmplYUVWriter::WriteNextFrame(mfxFrameSurface1 *pSurface)
     {
     case MFX_FOURCC_YV12:
     {
-        for (i = 0; i < (mfxU32)pInfo.CropH / 2; i++)
+        for (i = 0; i < ChromaH; i++)
         {
             MSDK_CHECK_NOT_EQUAL(
-                fwrite(pData.V + (pInfo.CropY * pData.Pitch / 2 + pInfo.CropX / 2) + i * pData.Pitch, 1, pInfo.CropW, dstFile),
+                fwrite(pData.V + (pInfo.CropY * pData.Pitch / 2 + pInfo.CropX / 2) + i * pData.Pitch, 1, ChromaW, dstFile),
```

Comment:
For YV12:
CropX and CropY should be left/top aligned. So if they are odd, CropX/2 and CropY/2 are right themselves. But in this case, the first row/column of U/V may be only corresponding with the first row/column of Y. And the 2nd row/column of U/V should be corresponding with the 2nd/3rd row/column Y. It may not match YV12 rule? Otherwise, we may have to do interpolation.
For NV12:
CropX should be OK because of U/V interleaved. But CropY should have the same issue.
For P010/P210:
They are similar with 8bit cases.

So I am not sure if we can assume CropX/CropY must not be odd?
In fact, my current case is the width/height of the clip is odd. So both CropX and CropY are zero.
Thanks.